### PR TITLE
add documentation and take out stable notification & restructure

### PR DESCRIPTION
**Pull request by britt6612**
- Takes out `stable` from new data and friends components 
- Add documentation to DataSummary
- Restructure how components are organized in site to be consistent with storybook

### CODE REVIEW

**Review by taysea — APPROVED** [review]
LGTM, thank you!

**Review by taysea — APPROVED** [review]
Thanks for adding `badge` documentation!

File: src/screens/Data.js
Comment (jcfilben) [inline src/screens/Data.js:1]: I think we need to remove the stable flag from this page

Comment (britt6612) [reply src/screens/Data.js:1]: on it

**Review by jcfilben — APPROVED** [review]
Looks good!

**Pull request merged**